<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="SuiteDDemo" duration-ms="16424" started-at="2022-08-23T21:10:15Z" finished-at="2022-08-23T21:10:32Z">
    <groups>
    </groups>
    <test name="Test" duration-ms="16424" started-at="2022-08-23T21:10:15Z" finished-at="2022-08-23T21:10:32Z">
      <class name="TestTest">
        <test-method status="PASS" signature="beforeTest()[pri:0, instance:TestTest@26ba2a48]" name="beforeTest" is-config="true" duration-ms="10" started-at="2022-08-24T05:10:15Z" finished-at="2022-08-24T05:10:15Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeTest -->
        <test-method status="PASS" signature="beforeClass()[pri:0, instance:TestTest@26ba2a48]" name="beforeClass" is-config="true" duration-ms="0" started-at="2022-08-24T05:10:15Z" finished-at="2022-08-24T05:10:15Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeClass -->
        <test-method status="FAIL" signature="test01()[pri:0, instance:TestTest@26ba2a48]" name="test01" duration-ms="16397" started-at="2022-08-24T05:10:15Z" finished-at="2022-08-24T05:10:32Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [鐧惧害涓�涓嬶紝浣犲氨鐭ラ亾] but found [百度一下，你就知道]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [鐧惧害涓�涓嬶紝浣犲氨鐭ラ亾] but found [百度一下，你就知道]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:190)
	at org.testng.Assert.assertEquals(Assert.java:200)
	at TestTest.test01(TestTest.java:20)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:178)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:92)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:96)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
	at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
	at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test01 -->
        <test-method status="PASS" signature="beforeMethod()[pri:0, instance:TestTest@26ba2a48]" name="beforeMethod" is-config="true" duration-ms="0" started-at="2022-08-24T05:10:15Z" finished-at="2022-08-24T05:10:15Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method status="PASS" signature="afterClass()[pri:0, instance:TestTest@26ba2a48]" name="afterClass" is-config="true" duration-ms="0" started-at="2022-08-24T05:10:32Z" finished-at="2022-08-24T05:10:32Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterClass -->
        <test-method status="PASS" signature="afterMethod()[pri:0, instance:TestTest@26ba2a48]" name="afterMethod" is-config="true" duration-ms="0" started-at="2022-08-24T05:10:32Z" finished-at="2022-08-24T05:10:32Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="PASS" signature="afterTest()[pri:0, instance:TestTest@26ba2a48]" name="afterTest" is-config="true" duration-ms="0" started-at="2022-08-24T05:10:32Z" finished-at="2022-08-24T05:10:32Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterTest -->
      </class> <!-- TestTest -->
    </test> <!-- Test -->
  </suite> <!-- SuiteDDemo -->
</testng-results>
